packages/wonder-stuff-sentry/src/kind-sentry-error.js

```diff
@@ -0,0 +1,177 @@
+// @flow
+import {KindError, Errors} from "@khanacademy/wonder-stuff-core";
+import type {Metadata} from "@khanacademy/wonder-stuff-core";
+import {sentryDataReducer} from "./sentry-data-reducer.js";
+import type {SentryData} from "./types.js";
+
+const EmptySentryData: $ReadOnly<SentryData> = Object.freeze({
+    tags: {},
+    contexts: {},
+    fingerprint: [],
+});
+
+/**
+ * Options for constructing a `KindError`.
+ */
+type Options = {|
+    /**
+     * An error responsible for the error being created.
+     *
+     * @type {?Error}
+     */
+    cause?: ?Error,
```

Comment:
Tiniest nit: I don't know why, but I'm feeling split on calling this `cause` versus `sourceError`. I can't explain why I like one better than the other, it's just a feeling thing. But for whatever reason, the word `cause` just makes my brain have to do, like, an extra two seconds of processing. I mean, it makes sense to use "cause" because of the language "caused by", but maybe it's just an iota too vague and overloaded, while "sourceError" is just fully unambiguous.